File: app/src/main/java/com/babedev/javapoetcodelab/sample2/Sample2Activity.java
package com.babedev.javapoetcodelab.sample2;

import android.app.Activity;
import android.os.Bundle;
import java.lang.Override;

class Sample2Activity extends Activity implements Sample2View {
  private Sample2Presenter mPresenter;

  @Override
  public void onCreate(Bundle savedInstanceState) {
    // TODO setContentView()
    mPresenter = new Sample2Presenter();
    mPresenter.attachView(this);
  }
}

File: app/src/main/kotlin/com/babedev/kotlinpoetcodelab/sample2/Sample2Activity.kt
package com.babedev.kotlinpoetcodelab.sample2

import android.app.Activity
import android.os.Bundle

class Sample2Activity : Activity, Sample2View {
  lateinit var mPresenter: Sample2Presenter

  override fun onCreate(savedInstanceState: Bundle) {
    super.onCreate(savedInstanceState)
    // TODO setContentView()
    mPresenter = Sample2Presenter()
    mPresenter.attachView(this)
  }
}
